geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/data/CreateSchemaCommand.scala

```diff
@@ -31,9 +33,9 @@ trait CreateSchemaCommand[DS <: GeoMesaDataStore[_, _, _ ,_]] extends DataStoreC
     lazy val sftString = SimpleFeatureTypes.encodeType(sft)
     logger.info(s"Creating '${params.featureName}' with spec '$sftString'. Just a few moments...")
 
-    if (ds.getSchema(sft.getTypeName) == null) {
+    if (try { ds.getSchema(sft.getTypeName) == null } catch { case _: IOException => true }) {
       ds.createSchema(sft)
-      if (ds.getSchema(sft.getTypeName) != null) {
+      if (try { ds.getSchema(sft.getTypeName) != null } catch { case _: IOException => false }) {
```

Comment:
there is a disconnect between how getSchema should handle non-existing type names. In our data stores, we return null - I believe we patterned this after the JDBCDataStore. In the ContentDataStore (which CassandraDataStore extends), it throws an IOException. This seems to be the correct implementation, according to the javadocs for DataStore. This check is to handle both the null return value or the IOException. Not sure if we should update our data stores to throw an exception...